shell.html

```diff
@@ -20,8 +20,8 @@ <h5 class="text-xl font-medium leading-tight mt-2">Loading</h5>
       <div id="canvas-container" class="hidden flex flex-col justify-center items-center">
         <canvas id="canvas" class="w-[768px] h-[768px]" oncontextmenu="event.preventDefault()" tabindex=-1 />
       </div>
-      <img id="pinball" width="1637" height="2048" src="../../data/pinball.jpg"/>
-      <img id="pinball-test" width="4000" height="3000" src="../../data/pinball-test.jpg"/>
+      <img id="pinball" width="1637" height="2048" src="../../data/pinball.jpg" style="display: none"/>
```

Comment:
This is fundamental, setting `display `to `none `otherwise the size of the image do not correspond to the effective pixel dimensions.